scss/application.scss

```diff
@@ -1,26 +1,10 @@
-@import url(http://fonts.googleapis.com/css?family=Roboto);
-
-// Colors
-$blue: #00ADEE;
-$green: #BED23C;
-$grey: #C4C4C4;
-
-// Sizes
-$nav-height: 90px;
-$footer-height: 30vh;
-$triangle-heigth: 90px;
-$triangle-width: 100px;
-$tech-logo-size: 100px;
-
-// Font sizes
-$font-big: 24px;
-$font-medium: 16px;
-$font-small: 12px;
+@import "variables/colors";
+@import "variables/fonts";
+@import "variables/sizes";
 
 body {
-  margin: 0px 0px 0px 0px;
-  overflow: hidden;
   font-family: 'Roboto',serif;
+  margin: 0px 0px 0px 0px;
```

Comment:
If you really really really need this, change to: `margin: 0;`

Reply:
I do :sweat_smile: